src/main/scala/ch/openolitor/core/RouteService.scala

```diff
@@ -445,13 +447,13 @@ trait DefaultRouteService extends HttpService with ActorReferences with BaseJson
     }
   }
 
-  protected def downloadAll(zipFileName: String, fileType: FileType, ids: Seq[FileStoreFileId]) = {
+  protected def downloadAll(pdfFileName: String, fileType: FileType, ids: Seq[FileStoreFileId]) = {
     ids match {
       case Seq()            => complete(StatusCodes.BadRequest)
       case Seq(fileStoreId) => download(fileType, fileStoreId.id)
       case list =>
         val refs = ids.map(id => FileStoreFileReference(fileType, id))
-        downloadAsZip(zipFileName, refs)
+        downloadMergedPDFs(pdfFileName, refs)
```

Comment:
If we just download the files, there is no possibility to choose PDF/ZIP?

Reply:
no, on the "download" option there is no menu. Do you think it makes sense to add one??

Reply:
Don't know. I guess it could as handy as in the moment of generating, no?

Reply:
done